tests/unit/test_cost.py

```diff
@@ -34,7 +34,9 @@ def test_costs(self, cut_off):
         # Construct Problem
         signal = "Voltage [V]"
         model.parameter_set.update({"Lower voltage cut-off [V]": cut_off})
-        problem = pybop.Problem(model, parameters, dataset, signal=signal, x0=x0)
+        problem = pybop.FittingProblem(
+            parameters, dataset, model=model, signal=signal, x0=x0
```

Comment:
Since model is not optional, we can just provide model
```suggestion
            parameters, dataset, model, signal=signal, x0=x0
```